src/query/mod.rs

```diff
@@ -50,6 +50,23 @@ impl fmt::Display for Timestamp {
     }
 }
 
+pub enum InfluxDbQueryTypes<'a> {
```

Comment:
Please add some rustdoc comments so it's clear what this is used for for users of this library.